testing/sdk_cmd.py

```diff
@@ -22,6 +23,7 @@ def fn():
         return fn()
 
 
+@retry(stop_max_attempt_number=5, stop_max_delay=60)
```

Comment:
One concern is that this is also called from **almost all** other tests, where we may have unintended side-effects from calling the same CLI command multiple times. 

With the retry logic here, we assume that either the return codes are consistent and represent the actual cluster state or the CLI commands idempotent and we don't mind calling them multiple times.